lib/ufolint/validators/plistvalidators.py

```diff
@@ -289,29 +289,29 @@ def run_ufolib_import_validation(self):
                 # glyphs_dir_list is a list of lists mapped to glyphs dir name,
                 # glyphs dir path
                 gs = GlyphSet(
-                    str(rel_dir_path), ufoFormatVersion=self.ufoversion, validateRead=True
+                    str(rel_dir_path),
+                    ufoFormatVersion=self.ufoversion,
+                    validateRead=True,
+                    expectContentsFile=True,
```

Comment:
For (my) future reference, this is the new parameter that was added in fonttools v4.18.0